specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/LinkedService.json

```diff
@@ -4060,6 +4060,63 @@
         "domain",
         "accessToken"
       ]
+    },
+    "ResponsysLinkedService": { 
+      "x-ms-discriminator-value": "Responsys", 
+      "description": "Responsys linked service.", 
+      "type": "object", 
+      "allOf": [ 
+        { 
+          "$ref": "#/definitions/LinkedService" 
+        } 
+      ], 
+      "properties": { 
+        "typeProperties": { 
+          "description": "Responsys linked service properties.", 
+          "x-ms-client-flatten": true, 
+          "$ref": "#/definitions/ResponsysLinkedServiceTypeProperties" 
+        } 
+      }, 
+      "required": [ 
+        "typeProperties" 
+      ] 
+    }, 
+    "ResponsysLinkedServiceTypeProperties": { 
+      "description": "Responsys linked service properties.", 
+      "properties": {
+        "endpoint": {
```

Comment:
nit: please check which of these properties should be marked `readonly`

Reply:
[Here](https://github.com/Azure/azure-rest-api-specs/blob/c31daf20a26d4ec8e247f4261b61fb77334d260d/specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-03-15-preview/definitions/MigrateSqlServerSqlMITask.json#L57)'s an example